src/footer.py

```diff
@@ -0,0 +1,3 @@
+while True:
```

Comment:
In which platforms is this loop needed?
As far as I remember, it is not really needed anywhere.
On the unix port you can run with `-i`, and on ESP32 / STM32 the event loop is handled by lvesp32/lvstm32 imports, and I don't think it's needed on the js port.

Reply:
Correct; in fact it will just freeze the tab/browser if you keep the loop.

Reply:
I did not know about the -i option to lv_micropython when running on the unix port.
You are right that the loop is not needed.
I removed it.